dataset/src/main/scala/frameless/TypedColumn.scala

```diff
@@ -506,6 +506,38 @@ abstract class AbstractTypedColumn[T, U]
   def contains[TT, W](other: ThisType[TT, U])(implicit ev: U =:= String, w: With.Aux[T, TT, W]): ThisType[W, Boolean] =
     typed(self.untyped.contains(other.untyped))
 
+  /** 'Starts with' test.
```

Comment:
Leftover? :D

Reply:
changed the docs a bit, should match the commonly used pattern now :)